src/main/java/org/javacs/InferConfig.java

```diff
@@ -283,14 +283,30 @@ private Path bazelOutputBase(Path bazelWorkspaceRoot) {
         }
     }
 
+    private String projectView(Path bazelWorkspaceRoot) {
+        Path projectFile = bazelWorkspaceRoot.resolve(".project");
+        String projectTargets = "...";
+        LOG.info("Searching for project file in " + projectFile.toString());
+        try {
+            if ( Files.exists(projectFile) ) {
+                projectTargets = String.join(" ", Files.readAllLines(projectFile));
+            }
+            LOG.info("Found projectTargets: " + projectTargets);
+        } catch (IOException exception) {
```

Comment:
Happy for suggestions on how to handle error here? Just log and let it go? Or does it need to be fatal?